backend/src/main/java/com/tradomator/model/Balance.java

```diff
@@ -0,0 +1,18 @@
+package com.tradomator.model;
+
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Data;
+import lombok.NoArgsConstructor;
+
+@AllArgsConstructor
+@NoArgsConstructor
+@Data
+@Builder
+public class Balance {
+
+    private String asset;
+    private String free;
```

Comment:
maybe `unlocked` ?

Reply:
point taken thanks. Will keep for now. As soon I build the mapper/ dot will adapt wording. For now i better keep like is given in external Api document.